FXP Rewrite (#682)
geostyler/geostyler-sld-parser: CHANGELOG.md

> # Version 5.0.0

Reviewer: I think examples would be very helpful to understand the breaking nature of these two changes.

> - TagValues will be casted to theire types by default

Reviewer: ```suggestion
- TagValues will be casted to their respective types by default
```

  - use `parseTagValue` option of parserOptions to change behaviour

> - Labels with whitespaces have to use `<![CDATA[ ]]>` when using whitespace

Reviewer: This is very odd. Is there no way of having this more convenient? Does this only apply to leading or dangling whitespace? If so the note should be more specific

> - Labels with whitespaces have to use `<![CDATA[ ]]>` when using whitespace

Reviewer: ```suggestion
- Labels with whitespaces have to use `<![CDATA[ ]]>`
```

# Version 5.0.0

> - TagValues will be casted to their respective types by default

Reviewer 2: I think it's `cast` not `casted`.

```suggestion
- TagValues will be cast to their respective types by default
```

> - TagValues will be casted to their respective types by default

Reviewer 2: Could you add a line that emphasizes the removal of the `forceCasting` option?
